apps/codeforafrica/src/lib/marked.js

```diff
@@ -0,0 +1,16 @@
+import DOMPurify from "dompurify";
+import { JSDOM } from "jsdom";
+import { marked as realMarked } from "marked";
+
+const { window } = new JSDOM("");
+const purify = DOMPurify(window);
```

Comment:
```suggestion
/* eslint-env browser */

import DOMPurify from "dompurify";
import { marked as realMarked } from "marked";

let global;
if (typeof window !== "undefined") {
  global = window;
} else {
  // lib/marked will only be used server-side hence require is fine
  // eslint-disable-next-line global-require
  const { JSDOM } = require("jsdom");
  global = new JSDOM("").window;
}
const purify = DOMPurify(global)
```

We can switch that `require` to `import` but support for it is still experimental in [Webpack 5](https://stackoverflow.com/questions/68339243/how-can-i-use-top-level-await-in-typescript-next-js).

Reply:
Ola @kilemensi  just curious, other than`import` being one of the new ways to perform imports, what other benefits will we get from using the experimental support?

Also, I'm not sure if the `import` is supported even on the experimental (Still can't use it locally despite trying all the experimental defined [here](https://webpack.js.org/configuration/experiments/#experiments))

Reply:
> what other benefits will we get from using the experimental support?
>

For us, nothing I can think of and [looking around](https://stackoverflow.com/questions/46677752/the-difference-between-requirex-and-import-x), I don't see much... but being a standard is a big deal and if possible we should try to stick to it @esirK.

Enabling it in `next.config.js` should work fine when running the app... Haven't tried looking at how to enable it under Jess given Jess doesn't use `next.config.js`

Reply:
@kilemensi  I've tried different options for this including suggestions e.g [this](https://jestjs.io/docs/ecmascript-modules), and suggestions [here](https://stackoverflow.com/questions/70725063/jest-wont-accept-top-level-awaits-with-nodejs16-typescript) but I don't seem to be making any progress. Any other pointers I can look at?

Reply:
Since we're already using Next.js 12 @esirK , I think our best bet is to switch `jest-config-commons-ui/next.js` package to use Next.js's built-in [configuration](https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler). It should load `next.config.js` even under Jest.